compose_testable_vignettes.R

```diff
@@ -46,9 +46,19 @@ rdt.cmd.head <- function(first, path)
     "include.configuration=TRUE,\n",
     "reload.state=", !first, ",\n",
     "block={\n\n",
+    "loadNamespace <- function(package, lib.loc = NULL,\n",
```

Comment:
Are you sure the code block will always have access to this definition? Just to be on the safer side, I kept it inside.